integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -280,8 +280,13 @@ public int getattr(String path, FileStat stat) {
       if (!mFileSystem.exists(turi)) {
         return -ErrorCodes.ENOENT();
       }
-      final URIStatus status = mFileSystem.getStatus(turi);
-
+      URIStatus status = mFileSystem.getStatus(turi);
+      if (!status.isCompleted()) {
+        if (!waitForFileCompleted(turi)) {
```

Comment:
Should we consider the mOpenFiles?

Reply:
Do you mean to check if it is in `mOpenFiles`? The file could be closed during the wait period.